workspace/welcome-to-xod/014-internet/patch.xodp

```diff
@@ -0,0 +1,168 @@
+{
+  "comments": [
+    {
+      "content": "# Internet\n\nThere is a large class of tasks, called Internet-Of-Things when the hardware needs an Internet connection to send or receive data from some server. For example, log the temperature and moisture for smart greenhouses, getting the current time, weather, work with XOD Cloud or other APIs, and so on.\n\nIn this chapter, we get the current date and time from the XOD Cloud API.\n\nFor this, there is the `xod-cloud/basics/now` node, which sends a request to the XOD Cloud, parses, and outputs the response as `datetime` type value.\n\nThis node has an input pin `INET`. This pin has to be connected with an internet provider. The Internet can be provided:\n\n- by the controller board\n  (such as ESP8266)\n- by an extension module\n  (such as W5500 Ethernet Shield)\n- **by your PC**. This trick is known as Internet tethering and works in simulation and debug modes",
+      "id": "HyGHq2IhU",
+      "position": {
+        "units": "slots",
+        "x": 1,
+        "y": 0
+      },
+      "size": {
+        "height": 6,
+        "units": "slots",
+        "width": 7
+      }
+    },
+    {
+      "content": "Note, we got a `select` node here to get the reasonable value until we get a result from the cloud. Otherwise, we would get a default datetime value, which is equal to `1970-01-01 00:00:00`, and it might be confusing.\n\n## Excercise\nLet's update the current datetime value by sending pulses.\n1. Place a `tweak-pulse` node.\n2. Link it to the `INIT` pin\n3. To make a value in the `watch` node proper, you need to provide a third possible value \"Waiting pulse\". Drag the handle on the right side of `select` node and add one more couple of input pins\n4. Bind `\"Waiting pulse\"` and \"On Boot\" in the latest pair of pins.\n5. Link `tweak-pulse` in the `S2` pin\n6. Simulate the patch and send pulse using `tweak-pulse` node.",
```

Comment:
Grammar

```suggestion
      "content": "Note, we’ve placed a `select` node here to get a reasonable value until we receive the result from the cloud. Otherwise, we would get a default datetime value, which is equal to `1970-01-01 00:00:00`, and it might be confusing.\n\n## Excercise\nUpdate the current datetime value by sending pulses.\n1. Place a `tweak-pulse` node.\n2. Link it to the `INIT` pin\n3. To make a value in the `watch` node properly, you need to provide the third possible value `\"Waiting pulse\"`. Drag the handle of the `select` node to the right to add one more pair of input pins\n4. Bind `\"Waiting pulse\"` and \"On Boot\" to the latest pair of pins.\n5. Link `tweak-pulse` to the `S2` pin\n6. Simulate the patch and send pulse using the `tweak-pulse` node.",
```